BoundedKnapsack/BoundedKnapsack.csproj

```diff
@@ -1,18 +1,40 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.Quantum.Sdk/0.12.20082513">
   <PropertyGroup>
+    <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <PlatformTarget>x64</PlatformTarget>
-    <IsPackable>false</IsPackable>
-    <RootNamespace>Quantum.Kata.BoundedKnapsack</RootNamespace>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="premerge\**" />
+    <EmbeddedResource Remove="premerge\**" />
+    <QsharpCompile Remove="premerge\**" />
+  </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.12.20072031" />
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.12.20072031" />
-    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.12.20072031" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
+    <PackageReference Include="Microsoft.Quantum.Katas" Version="0.12.20082513" />
+    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.12.20082513" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.12.20082513" />
+    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.12.20082513" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
   </ItemGroup>
+
+  <ItemGroup>
+    <QsharpCompile Include="Hints.md" />
+    <QsharpCompile Include="Hints.md" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
What is the purpose of these?

Reply:
Thank you for identifying the extraneous code. I think the code was generated by Visual Studio when I made project, and I wasn't been sure if it was essential to the kata or not.